dspace/config/crosswalks/DIM2DataCite.xsl

```diff
@@ -3,17 +3,20 @@
 <!--
     Document   : DIM2DataCite.xsl
     Created on : January 23, 2013, 1:26 PM
+    Updated on : November 26, 2015, 3:00 PM
     Author     : pbecker, ffuerste
     Description: Converts metadata from DSpace Intermediat Format (DIM) into
                  metadata following the DataCite Schema for the Publication and
-                 Citation of Research Data, Version 2.2
+                 Citation of Research Data, Version 3.1
 -->
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
                 xmlns:dspace="http://www.dspace.org/xmlns/dspace/dim"
-                xmlns="http://datacite.org/schema/kernel-2.2"
-                version="1.0">
+                xmlns="http://datacite.org/schema/kernel-3"
+                version="2.0">
     
     <!-- CONFIGURATION -->
+    <!-- Please add your DOI-Prefix and your namespace separator here (e.g. 10.5072/dspace-). -->
+    <xsl:variable name="prefix">10.5072/dspace-</xsl:variable>
```

Comment:
I agree that if we can fix it, we should. But if we can't fix it in time, it's better to put it in in its current form.